docs/make.js

```diff
@@ -17,51 +17,78 @@ const debug = (files, metalsmith, done) => {
 */
 
 const docs = articles => (files, metalsmith, done) => {
-  setImmediate(done)
+  setImmediate(done);
   Object.assign(metalsmith.metadata(), {
     docs: articles.map(name => {
-      const f = files[name + '.md']
-      f.isDoc = true
-      return f
-    }),
-  })
-}
+      const f = files[name + ".md"];
+      f.isDoc = true;
+      return f;
+    })
+  });
+};
 
 Metalsmith(__dirname)
   .metadata({
-    version: require('../package.json').version,
-    css: fs.readFileSync('../www/main.css', 'utf-8'),
+    version: require("../package.json").version,
+    css: fs.readFileSync("../www/main.css", "utf-8")
   })
-  .source('md/')
-  .destination('.')
+  .source("md/")
+  .destination(".")
   .clean(false)
-  .use(paths({
-    property: 'paths'
-  }))
-  .use(docs([
-
-    'index',
-    'getting-started',
-    'grammar',
-    'parser',
-    'tokenizers',
-    'tooling',
-    'how-to-grammar-good',
-    'using-in-frontend',
-    'glossary',
-
-  ]))
+  .use(
+    paths({
+      property: "paths"
+    })
+  )
+  .use(
+    docs([
+      "index",
+      "getting-started",
+      "grammar",
+      "parser",
+      "tokenizers",
+      "tooling",
+      "how-to-grammar-good",
+      "using-in-frontend",
+      "glossary"
+    ])
+  )
   //.use(debug)
-  .use(markdown({
-    smartypants: true,
-    gfm: true,
-  }))
-  .use(headings('h3'))
-  .use(layouts({
-    engine: 'handlebars',
-    default: 'template.html',
-  }))
+  .use(
+    markdown({
+      // accessToken: "github oath token goes here if being rate-limited"
```

Comment:
You can put a GitHub OAuth token here if you're being rate-limited while building.

Reply:
And yes, I know OAuth is misspelled.